docs/testing/sideload-office-addin-using-sideload-command.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Sideload Office Add-ins Using Sideload Command
```

Comment:
Titles use lowercase for non-proper nouns ("Sideload Office Add-ins using the sideload command")